Flank/flank
# feat: Add printing total run duration

Fixes #1666

## Test Plan
> How do we know the code works?

Total run duration is printed after Flank run with detailed log level set.

1. Set log level detailed
2. Run Flank
3. Check if you have time report at the end of run, ex:
```
Total run duration: 3m 11s
	- Preparation: 0m 6s
	- Running tests: 0m 6s
	- Executing matrices: 2m 50s
	- Generating reports: 0m 4s
```

## Checklist
